New example script: add new data to an existing dataset (#41)
quality-match/hari-client: docs/example_code/upload_new_data_to_existing_dataset/README.md

# Upload new data to an existing dataset

The script `upload_new_data_to_existing_dataset.py` demonstrates how to upload new data to an existing dataset that was previously created using the `quickstart.py` script.

> > :warning: <b>Disclaimer: In most cases it makes more sense to upload your data as a new dataset - especially when you've already done a lot of work with that dataset through annotation runs</b>

Reviewer: I would not add this, My upload crashed regularly and thus I need this functionality.

Author: makes sense - I've rephrased the case where it doesn't make too much sense. Hope it's clearer now.

Note: The script was written with hari_client version 2.0.3.

Things that are implemented in the script:

- add one new media
- add one new media object
- make sure that you're not uploading duplicate data
  - based on the provided back references on the new media and media object
- make sure that you're reusing already existing `initial_attributes` correctly

>   - by reusing the existing attribute ids and attribute types

Author: Because the backend expects you to have set that correctly.

# Upload new data to an existing dataset

The script `upload_new_data_to_existing_dataset.py` demonstrates how to upload new data to an existing dataset that was previously created using the `quickstart.py` script.

> > :warning: <b>Disclaimer: In the case that you've already did a lot of work with your dataset with annotation runs, we recommend that you create a new dataset and upload your new data there in order to keep a better overview.</b>

Reviewer 2: ```suggestion
> :warning: <b>Disclaimer: In the case that you've already used your dataset for annotation runs, we recommend that you create a new dataset and upload your new data there in order to keep a better overview.</b>
```
